test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -184,6 +184,78 @@ public void TestGreyscaleTransformImages()
             Done();
         }
 
+        [Fact]
+        public void TestGrayScaleInMemory()
```

Comment:
>TestGrayScaleInMemory [](start = 20, length = 21)

Need to add a testfor some other random type, such as Point and a custom transform that uses it from IEnumerable to IDV.